web/crux-ui/locales/en/notifications.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title": "Notifications",
```

Comment:
`title` is available as `common:notifications`
`title-notification` should be like:
`"notificationsName": "Notifications - {{name}}",`